.github/workflows/buildpack/check-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${GITHUB_TOKEN}" ] || [ -z "${GITHUB_ACTOR}" ] || [ -z "${GITHUB_DOCKER_REPOSITORY}" ]; then
+  echo "\${IMAGE_NAME}, \${IMAGE_VARIANT}, \${GITHUB_TOKEN}, \${GITHUB_ACTOR} and \${GITHUB_DOCKER_REPOSITORY} need to be defined."
+  exit 1
+fi
+
+VERSION=$(grep -e 'LABEL version=\".*\"' "scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}" | awk -F"\"" '{ print $2 }')
+echo "::set-env name=VERSION::${VERSION}"
+
+DOCKER_IMAGE_ID="$IMAGE_NAME-$IMAGE_VARIANT"
+if [ "${IMAGE_VARIANT}" == "emscripten" ]; then
+  EMSDK_VERSION=$(grep "emscripten/emsdk" scripts/docker/buildpack/Dockerfile.emscripten | cut -f 2 -d ' ' | cut -f 2 -d ':')
+  DOCKER_IMAGE_ID="${DOCKER_IMAGE_ID}-${EMSDK_VERSION}"
+fi
```

Comment:
Ah.. I thought that this is needed, because the old images seem to have the emsdk version encoded in the name. Yes, could just don't add it to simplify things.